lua/hunk/config.lua

```diff
@@ -27,6 +27,8 @@ local M = {
       mode = "nested",
       width = 35,
     },
+    --- Can be either `vertical` or `horizontal`
+    split_type = "vertical",
```

Comment:
I think I'd prefer this being called `layout`

Reply:
Was a bit unsure on the naming as well, `layout` is great.